configs/webworker/webpack.config.babel.js

```diff
@@ -4,7 +4,7 @@ const config = {
   target: 'webworker',
   entry: path.resolve(__dirname, './api-flow.js'),
   output: {
-    path: path.resolve(__dirname, '../../../dist/node/'),
```

Comment:
This caused other issues where running webworker and node in make would override node.